django/contrib/admin/utils.py

```diff
@@ -459,7 +460,7 @@ def display_for_value(value, empty_value_display, boolean=False):
 
     if boolean:
         return _boolean_icon(value)
-    elif value is None:
+    elif value in EMPTY_VALUES:
```

Comment:
We don't have a field here, so I'd use the safest option:
```suggestion
    elif value in (None, ""):
```

Reply:
In `display_for_value` is a check for `list` or `tuple`:

```
    elif isinstance(value, (list, tuple)):
        return ", ".join(str(v) for v in value)
```

`value` could be at least a `list` or `tuple`. And the above would turn the empty `list` or `tuple` again in an empty string (which to avoid is the whole point of this and the former PR). So I guess it is reasonable to use `EMPTY_VALUES` which is `(None, "", [], (), {})` here?

Reply:
Let's keep it for consistency.